### PR TITLE
feat: use dark/light mode syncing colorscheme format

### PR DESCRIPTION
**Pull request by nekowinston**

### CODE REVIEW

**Comment by nekowinston** [comment]
finally could confirm this is backwards compatible to stable.

File: build.ts [hunk 1]
```diff
@@ -1,147 +1,86 @@
...
+    "Ansi 13 Color": colors.pink,
+    "Ansi 14 Color": colors.teal,
+    "Ansi 15 Color": dark ? colors.surface1 : colors.subtext0,
+    "Background Color": colors.base,
+    "Foreground Color": colors.text,
+    "Link Color": colors.sky,
+    "Bold Color": colors.text,
+    "Cursor Color": colors.rosewater,
+    "Cursor Text Color": colors.base,
+    "Cursor Guide Color": { ...colors.text, alpha: 0.07 },
```
Comment (MatthiasPortzel) [inline build.ts:61]: This needed to be `"Alpha Component": 0.07`.

Comment (MatthiasPortzel) [inline build.ts:61]: This needed to be `"Alpha Component": 0.07`. https://github.com/catppuccin/iterm/blob/7a71cf49c88f9d729ffebeec9ddb9e83b09a0d00/colors/catppuccin-mocha.itermcolors#L39-L44

File: build.ts [hunk 2]
```diff
@@ -1,147 +1,86 @@
...
+    "Ansi 6 Color": colors.teal,
+    "Ansi 7 Color": dark ? colors.subtext1 : colors.surface2,
+    "Ansi 8 Color": dark ? colors.surface2 : colors.subtext0,
+    "Ansi 9 Color": colors.red,
+    "Ansi 10 Color": colors.green,
+    "Ansi 11 Color": colors.yellow,
+    "Ansi 12 Color": colors.blue,
+    "Ansi 13 Color": colors.pink,
+    "Ansi 14 Color": colors.teal,
+    "Ansi 15 Color": dark ? colors.surface1 : colors.subtext0,
```
Comment (MatthiasPortzel) [inline build.ts:54]: These two need to be switched since it was `isLatte ? colors.surface1 : colors.subtext0` before.

**Review by MatthiasPortzel — COMMENTED** [review]
PR review right-on-time.
Will put a PR with these changes as soon as I get a chance (i.e. not right now).

**Pull request merged**